File: langhuan-service/src/main/java/com/langhuan/model/pojo/RagChangeDocumentsReq.java
package com.langhuan.model.pojo;

import com.langhuan.model.domain.TRagFile;
import lombok.Data;

import java.util.List;

@Data
public class RagChangeDocumentsReq {
    private List<String> documents;
    private String documentId;
    private TRagFile ragFile;
}

File: langhuan-server/src/main/kotlin/com/langhuan/model/pojo/RagChangeDocumentsReq.kt
package com.langhuan.model.pojo

import com.langhuan.model.domain.TRagFile
import lombok.Data

@Data
data class RagChangeDocumentsReq(
    var documents: String? = null,
    var documentId: String? = null,
    var ragFile: TRagFile? = null
)
